services/core/shared/core/compat/sdk_v5/snapshotUtils.js

```diff
@@ -0,0 +1,110 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const path = require('path');
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
```

Comment:
Please consider renaming the variable to something like `execInShell` and update wherever relevant.
```suggestion
const execInShell = util.promisify(require('child_process').exec);
```